spec/cybersource/remote/integration_spec.rb

```diff
@@ -634,7 +634,7 @@
 
   describe 'with merchant descriptor' do
     before do
-      @properties << build_property('merchant_descriptor', {"name"=>"Ray Qiu", "contact"=>"6508883161"}.to_json)
+      @properties << build_property('merchant_descriptor', {"name"=>"Ray Qiu\u{c3}\u{bc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}\u{fc}", "contact"=>"6508883161"}.to_json)
```

Comment:
You need to have a patch that does not break names with unicode characters that are not in the ascii range you specified. Right 'Stéphane', André, ... would be broken... I think you fix is too simplistic